src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -66,6 +67,15 @@ export function PharmacyCard(props: PharmacyProps) {
       </Banner>
     );
   };
+
+  const isPopup = (organizationId: number) => organizationId === 25;
```

Comment:
You would need to update this to `organizationId: number | undefined`

I like the contextual information provided in the function name, though I personally use constants. The function is also pretty tiny, and could be written inline:

```
const POPUP_ORGANIZATION_ID = 25;
const popupWarningMarkup =props.organizationId === POPUP_ORGANIZATION_ID ? (
```